Assets/DefaultConfig.txt

```diff
@@ -23,6 +23,23 @@ Connections
             Title="WHO"
             CaptureUntil="^Total: [0-9]+ players"
           }
+          {
```

Comment:
Looks like this isn't a valid file, I think you need an extra { } around the whole trigger.

Reply:
You deleted the WHO spawn trigger?